3-extra/1-factorial.js

```diff
@@ -9,7 +9,11 @@
 */
 
 function factorial(input) {
-    // TODO
+    let total = 1;
+    for(let index=input;index>0;index--){
```

Comment:
Always remember to include spaces, they're not needed by the compiler or processor which reads the code, but it helps us read it better as developers 😅 

I'm not sure you actually completed this task, we can talk about it in the hawk channel if you want.